app/code/Magento/Review/Block/Adminhtml/Edit/Form.php

```diff
@@ -75,18 +75,24 @@ protected function _prepareForm()
         $review = $this->_coreRegistry->registry('review_data');
         $product = $this->_productFactory->create()->load($review->getEntityPkValue());
 
+        $formActionParams =['id' => $this->getRequest()->getParam('id'),
```

Comment:
@ravi-chandra3197 does this formatting looks good to you?

Reply:
Hello @orlangur
I have updated PR.